README.md

```diff
@@ -140,9 +140,17 @@ usrsctp-1.0-SNAPSHOT-7a8bc9a-linux-x86_64
 * The `sctp` module contains the Java library on top of the JNI code. 
 The jar built by this is what is intended to be used by other code.
 
+## Pre-required software to build
+`CMake` is required to build native libraries. 
+Native compilers `GCC`/`Clang`/`Visual C++` required to build native libraries.
+
 ### Building the jar files
 * Clone the project and initialize [`usrsctp`](https://github.com/sctplab/usrsctp) git submodule.
-* Run `mvn package -f pom.xml` (and `mvn install -f pom.xml` to install locally). It assumes that platform specific artifacts like `usrsctp` and `jniwrapper-native` are already available in Maven. To compile and install all artifacts locally run `mvn install -Pbuild-usrsctp -Pbuild-jnisctp -f pom.xml`, this will build native and `Java` code for current platform and install resulting artifacts to local repository.
+    ```
+    > git clone --recurse-submodules https://github.com/jitsi/jitsi-sctp.git jitsi-sctp
+    ```
+* Run `mvn install -Pbuild-usrsctp -Pbuild-jnisctp -f pom.xml` to build both `Java` and native artifacts for current platform and install them into local `Maven` repository.
+If you don't want to re-build native artifacts `usrsctp` and `jniwrapper-native` and prefer using pre-built version of them from `Maven` you may build and install `Java` artifacts via `mvn install -f pom.xml`.
```

Comment:
To make sure I'm understanding: the only 'pre-built' version from Maven is one the user would have had to build and install locally themselves, is that right?  Or are you thinking those could be published separately?

Reply:
How does this work exactly, if I build jitsi-videobridge on arm64 and your maven has only x86, does it detect that and rebuilds as needed or I still need to wait for you to publish to maven arm64 too?

The first option would be nicer imho, not sure if possible.

Reply:
> the only 'pre-built' version from Maven is one the user would have had to build and install locally themselves, is that right?

That would also work, but I meant slightly different thing.

> Or are you thinking those could be published separately?

I meant that native artifacts had to being built on each supported platform separately and in advance. These platform specific artifacts (`usrsctp` and `jniwrapper-native`) must be published to public maven repository for each supported platform. Then user may use those pre-built artifacts for his platform. And `jitsi-sctp` distributor might build so called `fat` jar, which will pack all pre-built native artifacts into single `fat` jar, which is suitable to use on any supported platform.

Reply:
> How does this work exactly, if I build jitsi-videobridge on arm64 and your maven has only x86, does it detect that and rebuilds as needed or I still need to wait for you to publish to maven arm64 too?

So if you are building `jitsi-videobridge` on `arm64` and official version has only `x86` you will have to build `jitsi-sctp` for `arm64` yourself: this could be done with `mvn install -Pbuild-usrsctp -Pbuild-jnisctp -f pom.xml`. But if there is `arm64` build machine I think it is doable to automate build for `arm64` and publishing to maven.